src/cli.rs

```diff
@@ -186,59 +213,71 @@ pub struct TextForamttingCliOptions {
         value_parser = value_parser!(u8).range(..16),
         num_args = 1..=6
     )]
+    #[merge(strategy = overwrite_vector)]
     pub text_colors: Vec<u8>,
     /// Use ISO 8601 formatted timestamps
     #[arg(long, short = 'z')]
+    #[merge(strategy = merge::bool::overwrite_false)]
     pub iso_time: bool,
     /// Which thousands SEPARATOR to use
     #[arg(long, value_name = "SEPARATOR", default_value = "plain", value_enum)]
+    #[merge(strategy = overwrite)]
     pub number_separator: NumberSeparator,
     /// Turns off bold formatting
     #[arg(long)]
+    #[merge(strategy = merge::bool::overwrite_false)]
     pub no_bold: bool,
 }
-#[derive(Clone, Debug, Args, PartialEq, Eq, Default)]
+#[derive(Clone, Debug, Args, PartialEq, Eq, Default, Merge, Deserialize, Serialize)]
 #[command(next_help_heading = "VISUALS")]
 pub struct VisualsCliOptions {
     /// Hides the color palette
     #[arg(long)]
+    #[merge(strategy = merge::bool::overwrite_false)]
     pub no_color_palette: bool,
     /// Hides the ascii art or image if provided
     #[arg(long)]
+    #[merge(strategy = merge::bool::overwrite_false)]
     pub no_art: bool,
     /// Use Nerd Font icons
     ///
     /// Replaces language chips with Nerd Font icons
     #[arg(long)]
+    #[merge(strategy = merge::bool::overwrite_false)]
     pub nerd_fonts: bool,
 }
 
-#[derive(Clone, Debug, Args, PartialEq, Eq, Default)]
+#[derive(Clone, Debug, Args, PartialEq, Eq, Default, Merge, Deserialize, Serialize)]
 #[command(next_help_heading = "DEVELOPER")]
 pub struct DeveloperCliOptions {
     /// Outputs Onefetch in a specific format
     #[arg(long, short, value_name = "FORMAT", value_enum)]
+    #[serde(skip)]
     pub output: Option<SerializationFormat>,
     /// If provided, outputs the completion file for given SHELL
     #[arg(long = "generate", value_name = "SHELL", value_enum)]
+    #[serde(skip)]
     pub completion: Option<Shell>,
 }
 
-#[derive(Clone, Debug, Args, PartialEq, Eq, Default)]
+#[derive(Clone, Debug, Args, PartialEq, Eq, Default, Merge, Deserialize, Serialize)]
 #[command(next_help_heading = "OTHER")]
 pub struct OtherCliOptions {
     /// Prints out supported languages
     #[arg(long, short)]
+    #[merge(strategy = merge::bool::overwrite_false)]
     pub languages: bool,
     /// Prints out supported package managers
     #[arg(long, short)]
+    #[merge(strategy = merge::bool::overwrite_false)]
     pub package_managers: bool,
 }
 
 impl Default for CliOptions {
     fn default() -> CliOptions {
         CliOptions {
             input: PathBuf::from("."),
+            config_path: Default::default(),
```

Comment:
```suggestion
            config_path: None,
```
Nitpick: I think an explicit `None` is a bit better, here.